static/js/components/B2BReceiptExplanation.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from "react"
+
+import Expandable from "./Expandable"
+
+type Props = {
+  className?: string
+}
+const B2BReceiptExplanation = ({ className }: Props) => (
+  <div className={`container b2b-explanation ${className ? className : ""}`}>
+    <div className="row">
+      <div className="col-lg-8">
```

Comment:
I know some or all of this is based on the already-existing styles and classes, but this component was rendering in ways that seemed odd to me at certain screen widths (in Chrome). When the screen dropped below 635px, the box stopped shrinking horizontally, so you would need to scroll to the right to expand/collapse it. I made the following changes to make it expand and shrink in ways that made sense to me:

1. Change this div's class to `col-lg-12`, which makes more sense since this is the only column in the row and I don't see any reason why it wouldn't take up the whole width
2. Change the css selector `.purchase-summary-block .b2b-explanation .col-lg-8` to instead target `.col-lg-12`, and get rid of the style definitions except for `padding`
    
    ```scss
    .purchase-summary-block {
      //...
      .b2b-explanation {
        .col-lg-12 {
          padding-left: 0;
        }
    ```

Here are screenshots with those changes applied:

![ss 2020-03-11 at 11 42 49 ](https://user-images.githubusercontent.com/14932219/76436531-bae07700-638e-11ea-9d90-20e49577a405.png)
![ss 2020-03-11 at 11 43 01 ](https://user-images.githubusercontent.com/14932219/76436533-bb790d80-638e-11ea-8c36-4180506e859d.png)
![ss 2020-03-11 at 11 43 06 ](https://user-images.githubusercontent.com/14932219/76436535-bb790d80-638e-11ea-8a31-dda8f5e5a409.png)
![ss 2020-03-11 at 11 43 27 ](https://user-images.githubusercontent.com/14932219/76436536-bb790d80-638e-11ea-8019-cd29cee7fc45.png)